src/components/Help/Steppers/UserMountsStepper.js

```diff
@@ -0,0 +1,108 @@
+import UserMountsInstructionsConfirmChanges from 'utils/Assets/UserMountsInstructionsConfirmChanges.gif';
+import UserMountsInstructionsExpansionFilter from 'utils/Assets/UserMountsInstructionsExpansionFilter.gif';
+
+function getStepContent(stepIndex) {
+    switch (stepIndex) {
+        case 0:
+        return (
+            <div>
+                <Typography>{USER_MOUNTS_STEPPER_INSTRUCTIONS[0]}</Typography>
+                <div className='box'><img src={UserMountsInstructionsSelectUser} alt="How to select."/></div>
+            </div>
+            
+        );
+        case 1:
+        return (
+            <div>
+                <Typography>{USER_MOUNTS_STEPPER_INSTRUCTIONS[1]}</Typography>
+                <div className='box'><img src={UserMountsInstructionsChangeOwned} alt="How to edit."/></div>
+            </div>
+        );
+        case 2:
+        return (
+            <div>
+                <Typography>{USER_MOUNTS_STEPPER_INSTRUCTIONS[2]}</Typography>
+                <div className='box'><img src={UserMountsInstructionsConfirmChanges} alt="How to edit."/></div>
+            </div>
+        );
+        case 3:
+        return (
+            <div>
+                <Typography>{USER_MOUNTS_STEPPER_INSTRUCTIONS[3]}</Typography>
+                <div className='box'><img src={UserMountsInstructionsExpansionFilter} alt="How to change expansion."/></div>
+            </div>
+        );
+        case 4:
+        return (
+            <div>
+                <Typography>{USER_MOUNTS_STEPPER_INSTRUCTIONS[4]}</Typography>
+                <div className='box'><img src={UserMountsInstructionsExpansionFilter} alt="How to search."/></div>
+            </div>
+        );
+        default:
+        return 'Unknown stepIndex';
+    }
+}
+
+export const UserMountsStepper = () => {
+
+
+const [activeStep, setActiveStep] = React.useState(0);
+const steps = USER_MOUNTS_STEPPER_STEPS;
+
+const handleNext = () => {
+    setActiveStep((prevActiveStep) => prevActiveStep + 1);
+};
+
+const handleBack = () => {
+    setActiveStep((prevActiveStep) => prevActiveStep - 1);
+};
+
+const handleReset = () => {
+    setActiveStep(0);
+};
+
+return (
+  <div>
+    <Stepper activeStep={activeStep} alternativeLabel>
+      {steps.map((label) => (
+        <Step key={label}>
+          <StepLabel>{label}</StepLabel>
+        </Step>
+      ))}
+    </Stepper>
+    <div>
+      {activeStep === steps.length ? (
+        <div>
+          <Typography>All steps completed</Typography>
+          <Button onClick={handleReset}>Reset</Button>
+        </div>
+      ) : (
+        <div>
+          {getStepContent(activeStep)}
+          <div>
+            <Button disabled={activeStep === 0} onClick={handleBack}>
+              Back
+            </Button>
+            <Button variant='contained' color='primary' onClick={handleNext}>
+              {activeStep === steps.length - 1 ? 'Finish' : 'Next'}
```

Comment:
Please put hard-coded strings into constants. We might want to separate the constants for this `Help` page into another file.

Reply:
HelpConstants made, all free standing strings relocated